test/jdk/javax/swing/JTable/KeyBoardNavigation.java

```diff
@@ -273,21 +273,16 @@ public static String getOSSpecificInstructions() {
                 FN+Up Arrow/FN+Down Arrow - Deselect current selection;
                                    scroll up/down one JViewport view;
                                    first visible cell in current column gets focus
-                Control-FN+Up Arrow/FN+Down Arrow - Deselect current selection;
-                                                    move focus and view to
-                                                    first/last cell in current row
-                F2 - Allows editing in a cell containing information without
+                fn - Allows editing in a cell containing information without
```

Comment:
> 2. For the instruction mentioned at L273, when `FN+Down Arrow key` is pressed, viewport gets scroll down but I can see the cell selected is not the first visible cell in stead it is the first cell of last visible row.
>    Does this needs some rephrasing too?
Its actually last row's cell. If one cell is selected, it'll that cell column. If multiple cells are selected, it's first visible cell either first/last. Since the reporter has not raised any concern about this one, lets leave it that way.